packages/webapp/pages/onboarding.tsx

```diff
@@ -84,12 +101,52 @@ interface HeaderProps {
   showOnboardingPage: boolean;
   isOnboardingV3: boolean;
   setAuth: Dispatch<SetStateAction<AuthProps>>;
+  onClickNext: () => void;
+  isFiltering: boolean;
 }
 const Header = ({
   showOnboardingPage,
   isOnboardingV3,
+  isFiltering,
   setAuth,
+  onClickNext,
 }: HeaderProps) => {
+  const onboardingV4 = useFeature(feature.onboardingV4);
+  const isMobile = !useMedia([tablet.replace('@media ', '')], [true], false);
+  const isTablet = !useMedia([laptop.replace('@media ', '')], [true], false);
+
+  const getImage = () => {
+    if (isMobile) {
+      return cloudinary.feed.bg.mobile;
+    }
+
+    return isTablet ? cloudinary.feed.bg.tablet : cloudinary.feed.bg.laptop;
+  };
+
+  if (isFiltering && onboardingV4 === OnboardingV4.V4) {
+    return (
+      <header className="flex sticky top-0 z-3 justify-center mb-10 w-full backdrop-blur-sm">
+        <img
+          className="absolute top-0 right-0 left-0 w-full pointer-events-none"
+          src={getImage()}
```

Comment:
Not really urgent, but maybe we can consider using srcSet instead?
It will auto-handle this scenario and avoid having to manually import/re-render stuff?

It can be a separate backlog ticket for all I care, would love to hear your thoughts though.

Reply:
Yes, we use this on few places already, I added backlog ticket [WT-1932](https://dailydotdev.atlassian.net/browse/WT-1932)